### PR TITLE
fix some appimage issues building with qt 5.15.2

### PR DESCRIPTION
**Pull request by DioEgizio**
some users are having weird scaling issues since we're using qt 5.12.8 for the appimage
also removes redundant stuff

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -88,8 +96,8 @@ jobs:
         if: runner.os != 'Windows'
         uses: urkle/action-get-ninja@v1
 
-      - name: Prepare AppImage (Linux)
```
Comment (txtsd) [inline .github/workflows/build.yml:91]: Keep the `(Linux)` pls. It looks nice in the workflow views.

Comment (DioEgizio) [reply .github/workflows/build.yml:91]: opz, that was a mistake, fixing now

**Comment by txtsd** [comment]
Will this make it not run on distros that have older Qt?

**Comment by DioEgizio** [comment]
No, just updates bundled Qt on appimage (system and portable are still built with Qt 5.12)

**Comment by DioEgizio** [comment]
what the hell is launchpad dying?

**Comment by DioEgizio** [comment]
btw yesterday's issues now look solved, i guess launchpad was having issues?

**Review by txtsd — APPROVED** [review]
Looks and runs fine to me, but I can't test the scaling.

**Comment by DioEgizio** [comment]
I can't test either, but the user who had this issue said this fix works

**Comment by txtsd** [comment]
Okay good!

**Review by ZekeZDev — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]

**Pull request merged**